### PR TITLE
Add MIT License

### PR DESCRIPTION
**Pull request by logan2d**
Pull request to add MIT license per @devinsays

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 devonsays
```
Comment (devinsays) [inline LICENSE.md:3]: Can you update this from "devonsays" to "Devin Price"?

**Review by devinsays — CHANGES_REQUESTED** [review]

**Comment by logan2d** [comment]
Yes

File: LICENSE.md (hunk 1 above)
Comment (logan2d) [reply LICENSE.md]: change made and committed

**Review by devinsays — APPROVED** [review]

**Comment by devinsays** [comment]
@logan2d Cool, merged. I also updated this project to Laravel 8 and updated most npm dependencies over the weekend.

**Pull request merged**